ocs_ci/deployment/baremetal.py

```diff
@@ -706,6 +706,16 @@ def clean_disk():
             logger.info(out)
 
     for worker in workers:
+        cmd = """lsblk --all --noheadings --output "KNAME,PKNAME,TYPE,MOUNTPOINT" --json"""
+        out = ocp_obj.exec_oc_debug_cmd(node=worker.name, cmd_list=[cmd])
+        disk_to_ignore_cleanup_raw = json.loads(str(out))
+        disk_to_ignore_cleanup_json = disk_to_ignore_cleanup_raw["blockdevices"]
+        for disk_to_ignore_cleanup in disk_to_ignore_cleanup_json:
+            if disk_to_ignore_cleanup["mountpoint"] == "/boot":
+                logger.info(
+                    f"Ignorning disk {disk_to_ignore_cleanup['pkname']} for cleanup"
+                )
+                selected_disk_to_ignore_cleanup = disk_to_ignore_cleanup["pkname"]
```

Comment:
can we break here since we will have single boot partition

Reply:
boot disk will always have `/boot` in part and that particular disk will be skipped

Reply:
once you found boot disk, you can break the for loop

Reply:
yes can be done will try to use it

Reply:
Done